presto-docs/src/main/sphinx/functions/json.rst

```diff
@@ -22,12 +23,22 @@ Cast to JSON
         SELECT CAST(ARRAY[1, NULL, 456] AS JSON); -- JSON '[1,null,456]'
         SELECT CAST(ARRAY[ARRAY[1, 23], ARRAY[456]] AS JSON); -- JSON '[[1,23],[456]]'
         SELECT CAST(MAP(ARRAY['k1', 'k2', 'k3'], ARRAY[1, 23, 456]) AS JSON); -- JSON '{"k1":1,"k2":23,"k3":456}'
+        SELECT CAST(MAP(ARRAY['k1', 'k2', 'k3'], ARRAY[1, 23, 456]) AS JSON); -- JSON '{"k1":1,"k2":23,"k3":456}'
+        SELECT CAST( -- JSON '[123,"abc",true]'
```

Comment:
Can we fit this on one line if we change "value1" to "v1" etc.? I think that would be easier to read, since it lines up with the other examples.